version-catalog/README.md

```diff
@@ -0,0 +1,217 @@
+# SpineVersionCatalog
+
+Represents a set of dependencies used in Spine-related projects. It's assembled 
+and published in a form of Gradle's [Version Catalog](https://docs.gradle.org/current/userguide/platforms.html#sec:sharing-catalogs).
+
+
+## Usage example
+
+In order to use this catalog, one should perform the following:
+
+ 1. Put `spine-version-catalog` library on a classpath of `settings.gradle.kts` file.
+ 2. Create a new version catalog. `libs` is a conventional name to go with.
+ 3. Apply `SpineVersionCatalog` to a newly created catalog.
+
+Below is an example of how to obtain this catalog in the project.
+
+In `settings.gradle.kts` file of the project:
+
+```kotlin
+import io.spine.internal.catalog.SpineVersionCatalog
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("io.spine.internal:spine-version-catalog:2.0.0")
+    }
+}
+
+dependencyResolutionManagement {
+    versionCatalogs {
+        create("libs") {
+            SpineVersionCatalog.useIn(this)
+        }
+    }
+}
+```
+
+
+## Adding a new dependency to the catalog
+
+In order to add a new dependency to this catalog, perform the following steps:
+
+ 1. Go to `catalog` module.
+ 2. Open `io.spine.internal.catalog.entry` package.
+ 3. Create a new file there, which contains an object declaration, named after 
+a dependency, that is being added.
```

Comment:
Please adjust the text wrapping so that vertical line of item numbers is not broken by wrapped text.